queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
+SELECT name FROM animals WHERE neutered = true AND escape_attemps < 3;
+SELECT date_of_birth FROM animals WHERE name NOT IN ('Agumon','Pikachu');
```

Comment:
- [ ] Great work so far but I guess you misunderstood this query because you are supposed to get the weight of Agumon and Pikachu but I guess you excluded them 😄 Kindly fix this issue to make your queries work as expected 👍